packs/data/feat-effects.db/effect-elemental-assault-dualborn.json

```diff
@@ -0,0 +1,93 @@
+{
+    "_id": "R7eaWfY4I0Vs23VU",
+    "img": "systems/pf2e/icons/spells/elemental-zone.webp",
+    "name": "Effect: Elemental Assault (Dualborn)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>You shroud your arms and held weapons in elemental magic. Choose one element. Until the end of your next turn, your Strikes deal an additional [[/r 1d6]] damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>\n<p>As a dualborn, you can only use this effect with your chosen elements.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.TraitAir",
+                        "predicate": {
+                            "any": [
+                                "dualbornFirst:air",
+                                "dualbornSecond:air"
```

Comment:
Actually, we can get these all in on a single effect instead of splitting it into two. Let's fix up dualborn first though. We can do a predicate that looks like

```
"predicate": {
    "not": [{
        "and": ["feat:dualborn", {
            "not": ["dualborn:air"]
        }]
    }]
}
```

I think that will work anyways, may need to check the formatting.